rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/frame/SetupFrame.kt

```diff
@@ -64,7 +65,9 @@ internal class SetupFrame(
 
     override fun StringBuilder.appendSelf() {
         append("\nVersion: ").append(version.toString()).append(" Honor lease: ").append(honorLease).append("\n")
-        append("Keep alive: interval=").append(keepAlive.interval).append(", max lifetime=").append(keepAlive.maxLifetime).append("\n")
+        append("Keep alive: interval=").append(keepAlive.intervalMillis)
+            .append(" ms, max lifetime=").append(keepAlive.maxLifetimeMillis)
```

Comment:
suggest splitting this like the other lines. "max lifetime" starting on it's own line, this is debug code effectively so not performance critical.